<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chromatic-cute</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/res/picture/splash.png</normaloff>:/res/picture/splash.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QStackedWidget" name="m_stacked_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="CHomeWidget" name="m_page_home"/>
      <widget class="CPuzzleWidget" name="m_page_game"/>
      <widget class="CCreditsWidget" name="m_page_credits"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPuzzleWidget</class>
   <extends>QWidget</extends>
   <header>CPuzzleWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CHomeWidget</class>
   <extends>QWidget</extends>
   <header>CHomeWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CCreditsWidget</class>
   <extends>QWidget</extends>
   <header>CCreditsWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
